### PR TITLE
app-emulation/crun: new ebuild ( 0.10.5 )

### PR DESCRIPTION
**Pull request by dmolik**
Had some issues with python build dependencies, going to try again.

Package-Manager: Portage-2.3.79, Repoman-2.3.18
Signed-off-by: Dan Molik <dan@danmolik.com>
Tested-by: Dan Molik <dan@danmolik.com>

### CODE REVIEW

**Comment by dmolik** [comment]
Updated to reflect latest crun release

File: app-emulation/crun/crun-0.10.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		$(use_enable bpf) \
+		$(use_enable caps) \
+		$(use_enable seccomp) \
+		$(use_enable systemd) \
+		$(usex static-libs '--enabled-shared  --enabled-static' '--enable-shared --disable-static' '' '')
+}
+
+src_compile() {
+	pushd libocispec || die
+	emake || die
```
Comment (mrueg) [inline app-emulation/crun/crun-0.10.6.ebuild]: No || die needed

File: app-emulation/crun/crun-0.10.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	popd
```
Comment (mrueg) [inline app-emulation/crun/crun-0.10.6.ebuild]: Add || die here

File: app-emulation/crun/crun-0.10.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	emake crun
+	if use doc ; then
+		emake crun.1
+	fi
+}
+
+src_install() {
+	pushd libocispec || die
+	emake "DESTDIR=${D}" install-exec || die
```
Comment (mrueg) [inline app-emulation/crun/crun-0.10.6.ebuild]: no || die needed with emake

File: app-emulation/crun/crun-0.10.6.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit python-any-r1
+
+DESCRIPTION="A fast and low-memory footprint OCI Container Runtime fully written in C"
+HOMEPAGE="https://github.com/containers/crun"
+SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz"
```
Comment (mrueg) [inline app-emulation/crun/crun-0.10.6.ebuild]: Why use releases instead of archive here?

Comment (dmolik) [reply app-emulation/crun/crun-0.10.6.ebuild]: Giuseppe maintains a package called libocispec in a seperate repo, but includes it here as a submodule, as part of `make dist` it's include in the source release. On a side note I have yet to see another package use libocispec.

**Comment by mrueg** [comment]
Thanks, in as ccb71f94b4eb8d3a9d3cb25687f999c73d8a64e0

**Pull request closed without merging**